src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetadataGetIndexInfoTest.java

```diff
@@ -0,0 +1,174 @@
+            boolean hasNonClusteredIndex = false;
+            boolean hasColumnstoreIndex = false;
+            
+			String query = 
+		                "SELECT " +
+		                "    db_name() AS CatalogName, " +
+		                "    sch.name AS SchemaName, " +
+		                "    t.name AS TableName, " +
+		                "    i.name AS IndexName, " +
+		                "    i.type_desc AS IndexType, " +
+		                "    i.is_unique AS IsUnique, " +
+		                "    c.name AS ColumnName, " +
+		                "    ic.key_ordinal AS ColumnOrder " +
+		                "FROM " +
+		                "    sys.indexes i " +
+		                "INNER JOIN " +
+		                "    sys.index_columns ic ON i.object_id = ic.object_id AND i.index_id = ic.index_id " +
+		                "INNER JOIN " +
+		                "    sys.columns c ON ic.object_id = c.object_id AND ic.column_id = c.column_id " +
+		                "INNER JOIN " +
+		                "    sys.tables t ON i.object_id = t.object_id " +
+		                "INNER JOIN " +
+		                "    sys.schemas sch ON t.schema_id = sch.schema_id " +
+		
+		                "WHERE t.name = '" + table + "' " +
+		                      "AND sch.name = '" + schema + "' " +
+		                "ORDER BY " +
+		                "    t.name, i.name, ic.key_ordinal;";
+			rs2 = stmt.executeQuery(query);
+			
+            while (rs1.next() && rs2.next()) {
+                String indexType = rs1.getString("IndexType");
+                String indexName = rs1.getString("IndexName");
+                
+                assertEquals(rs1.getString("CatalogName"), rs2.getString("CatalogName"));
+                assertEquals(rs1.getString("SchemaName"), rs2.getString("SchemaName"));
+                assertEquals(rs1.getString("TableName"), rs2.getString("TableName"));
+                assertEquals(indexName, rs2.getString("IndexName"));
+                assertEquals(indexType, rs2.getString("IndexType"));
+                assertEquals(rs1.getString("IsUnique"), rs2.getString("IsUnique"));
+                assertEquals(rs1.getString("ColumnName"), rs2.getString("ColumnName"));
+                assertEquals(rs1.getString("ColumnOrder"), rs2.getString("ColumnOrder"));
+
+                if (indexType.contains("COLUMNSTORE")) {
+                    hasColumnstoreIndex = true;
+                } else if (indexType.equals("CLUSTERED")) {
+                    hasClusteredIndex = true;
+                } else if (indexType.equals("NONCLUSTERED")) {
+                    hasNonClusteredIndex = true;
+                }
+            }
+
+            assertTrue(hasColumnstoreIndex, "COLUMNSTORE index not found.");
+            assertTrue(hasClusteredIndex, "CLUSTERED index not found.");
+            assertTrue(hasNonClusteredIndex, "NONCLUSTERED index not found.");
+        }
+    }
+    
+    @Test
+    public void testGetIndexInfoCaseSensitivity() throws SQLException, SQLServerException {

```

Comment:
Exception not needed.
```suggestion
    public void testGenIndexInfoCaseSensitivity() throws SQLException {
```